server/routes/search/handlers/caSearch.ts

```diff
@@ -37,12 +38,10 @@ export default class CaSearch {
       }
     } else {
       const allPrisons = await this.prisonerService.getPrisons()
```

Comment:
do you need to do this api call?

Reply:
Ah it's specifically for the caseload switcher which we don't show on the search page. I'll removed this call.